python/lsst/pipe/tasks/functors.py

```diff
@@ -479,7 +480,13 @@ def __call__(self, data, **kwargs):
                     )
                     valDict[k] = f._func(subdf)
                 except Exception as e:
-                    self.log.error("Exception in %s call: %s: %s", self.name, type(e).__name__, e)
+                    self.log.error(
```

Comment:
Are you sure you don't want to use `self.log.exception` here and automatically report the stack trace? If someone sees an exception like this inside the functor code will they be able to work out where their code triggered it?

Reply:
I am not sure about that, I just changed this to work better than it was working.  How does one use `self.log.exception`?

Reply:
Same as `self.log.error` except it automatically includes the exception information.